buildscripts/executeSeleniumTestsReleaseBuild.sh

```diff
@@ -0,0 +1,198 @@
+echo Build image is $UX_ASPECTS_BUILD_IMAGE_NAME:$UX_ASPECTS_BUILD_IMAGE_TAG_LATEST
+echo UID is $UID
+echo GROUPS is $GROUPS
+echo USER is $USER
+echo PWD is $PWD
+echo HOME is $HOME
+echo Displaying groups
+groups
+echo Displaying id
+id
+
+HOME_FOLDER="/home/UXAspectsTestUser/UXAspectsTestsReleaseBuild"
+echo HOME_FOLDER is $HOME_FOLDER
+
+# Define a function to build a specified Docker image.
+docker_image_build()
+{
+    DOCKER_IMAGE_ID=`docker images | grep $UX_ASPECTS_BUILD_IMAGE_NAME | grep $UX_ASPECTS_BUILD_IMAGE_TAG_LATEST | awk '{print $3}'`
+    echo ID for $UX_ASPECTS_BUILD_IMAGE_NAME:$UX_ASPECTS_BUILD_IMAGE_TAG_LATEST image is $DOCKER_IMAGE_ID
+    if [ -z "$DOCKER_IMAGE_ID" ] ; then
+        # Create the docker image
+        cd $HOME_FOLDER/ux-aspects/docker
+        echo Building the image
+        docker build -t $UX_ASPECTS_BUILD_IMAGE_NAME:$UX_ASPECTS_BUILD_IMAGE_TAG_LATEST \
+            --build-arg http_proxy=$HttpProxy \
+            --build-arg https_proxy=$HttpsProxy \
+            --build-arg no_proxy="localhost, 127.0.0.1" \
+            --no-cache .
+        DOCKER_IMAGE_ID=`docker images | grep $UX_ASPECTS_BUILD_IMAGE_NAME | grep $UX_ASPECTS_BUILD_IMAGE_TAG_LATEST | awk '{print $3}'`
+        echo ID for new $UX_ASPECTS_BUILD_IMAGE_NAME:$UX_ASPECTS_BUILD_IMAGE_TAG_LATEST image is $DOCKER_IMAGE_ID
+    fi
+}
+
+# Define a function to run a specified Docker image. The job's workspace will be mapped to /workspace in the container.
+# The container will run using the UID of the user executing the job.
+docker_image_run()
+{
+    DOCKER_IMAGE_ID=`docker images | grep $UX_ASPECTS_BUILD_IMAGE_NAME | grep $UX_ASPECTS_BUILD_IMAGE_TAG_LATEST | awk '{print $3}'`
+    echo ID for $UX_ASPECTS_BUILD_IMAGE_NAME:$UX_ASPECTS_BUILD_IMAGE_TAG_LATEST image is $DOCKER_IMAGE_ID
+    if [ -z "$DOCKER_IMAGE_ID" ] ; then
+        echo Image $UX_ASPECTS_BUILD_IMAGE_NAME:$UX_ASPECTS_BUILD_IMAGE_TAG_LATEST does not exist!
+    else
+        echo Calling docker run ... "$@"
+		docker run -d -it \
+			--cidfile="$PWD/ContainerIDReleaseBuild" \
+			--volume "$PWD":/workspace:rw --workdir /workspace \
+			--user $UID:$GROUPS \
+			-e "http_proxy=$HttpProxy" \
+			-e "https_proxy=$HttpsProxy" \
+			-e "no_proxy=localhost, 127.0.0.1" \
+			-p 4001:4001 \
+			$UX_ASPECTS_BUILD_IMAGE_NAME:$UX_ASPECTS_BUILD_IMAGE_TAG_LATEST \
+			"$@"
+    fi
+}
+
+echo Before changing to ux-aspects folder
+echo PWD IS $PWD
+echo Listing this folder
+ls -al
+echo Listing the ux-aspects folder
+ls -al ux-aspects
+echo Listing the $HOME_FOLDER/ux-aspects folder
+ls -al $HOME_FOLDER/ux-aspects
+cd $HOME_FOLDER/ux-aspects
+
+# Update connect.js to allow building of the documentation. Publish on port 4001.
+echo Updating grunt/connect.js
+sed -i.bak 's/hostname: '"'"'localhost'"'"'/hostname: '"'"'0.0.0.0'"'"'/' grunt/connect.js
+sed -i.bak 's/livereload: true/livereload: false/g' grunt/connect.js
+sed -i.bak 's/4000/4001/g' grunt/connect.js
+
```

Comment:
I would recommend creating a "connect:selenium_release" grunt task with these parameters already populated, to avoid possible replace errors.